GameData/Kerbalism/Parts/LifeSupport/kerbalism-lifesupportmodule.cfg

```diff
@@ -0,0 +1,189 @@
+// ============================================================================
+// kerbalism-lifesupportmodule.cfg by APlayer updated by PiezPiedPy
+// Kerbalism external Life-Support module
+//   Adds an external life support process part to Kerbalism.
+// ============================================================================
+
+
++PART[kerbalism-chemicalplant]:NEEDS[Kerbalism,ProfileDefault]:BEFORE[Kerbalism]
+{
+  @name = kerbalism-lifesupportmodule
+  @title = External ECLSS Module
+  @category = Utility
+  @description = Seeing the lack of ECLSS redundancy, mission control urgently ordered us to convert this chemical plant to an external ECLSS module. Guaranteed to explode only 147.28% as often as the original.
+  %tags = _kerbalism
+  
+  MODULE:NEEDS[FeatureReliability]
+  {
+    name = Reliability
+    type = ProcessController
+    title = ECLSS
+    redundancy = Life Support
+    repair = true // anyone
+    mtbf = 72576000 // 8y
+    extra_cost = 2.5
+    extra_mass = 0.1
+  }
+  
+  MODULE
+  {
+    name = ProcessController
+    resource = _Scrubber
+    title = Scrubber
+    capacity = 2
+    running = true
+  }
+
+  MODULE
+  {
+    name = ProcessController
+    resource = _PressureControl
+    title = Pressure control
+    capacity = 2
+    running = true
+  }
+
+  MODULE
+  {
+    name = ProcessController
+    resource = _HumidityController
+    title = Humidity controller
+    capacity = 2
+    running = true
+  }
+
+  MODULE
+  {
+    name = ProcessController
+    resource = _WaterRecycler
+    title = Water recycler
+    capacity = 2
+    running = true
+  }
+
+  MODULE
+  {
+    name = ProcessController
+    resource = _WasteProcessor
+    title = Waste processor
+    capacity = 2
+    running = true
+  }
+
+  MODULE
+  {
+    name = ProcessController
+    resource = _MonopropFuelCell
+    title = Monoprop fuel cell
+    capacity = 2
+    running = true
+  }
+
+  @MODULE[ProcessController],*
+  {
+    @capacity *= 1.67
+  }
+
+  MODULE
```

Comment:
Same here, this duplicated config is begging to be be forgotten in a future update.

Reply:
nothing to see here, move along 🤣